pilot/pkg/networking/core/v1alpha3/listener_address.go

```diff
@@ -51,11 +52,15 @@ func getActualWildcardAndLocalHost(node *model.Proxy) (string, string) {
 	return WildcardIPv6Address, LocalhostIPv6Address
 }
 
-func getPassthroughBindIP(node *model.Proxy) string {
-	if node.SupportsIPv4() {
-		return InboundPassthroughBindIpv4
+func getPassthroughBindIPsForDualStack(ipMode model.IPMode) []string {
+	ptBindings := ptBindIPs[ipMode]
+	if len(ptBindings) == 0 {
```

Comment:
how can this happen?

Reply:
Hi @howardjohn, yeah, it may happen if there is no calling `DiscoverIPMode()` for a proxy.